source/utilities.tex

```diff
@@ -7320,12 +7340,12 @@
 contains an object of type \tcode{T}.
 
 \pnum
-A program
-that instantiates the definition of template \tcode{expected<T, E>}
-for a reference type, a function type, or
-for possibly cv-qualified types \tcode{in_place_t}, \tcode{unexpect_t}, or
-a specialization of \tcode{unexpected} for the \tcode{T} parameter
-is ill-formed.
+A type \tcode{T} is a \term{valid value type for \tcode{expected}}, if
```

Comment:
N.B. as per https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#latex-source-code-formatting we try to use "semantic line breaks" to minimise the size of diffs when we make later edits.

In this line, I would have put the line break after the comma, with the "if" on the next line. And in the line below, break at the end of the sentence, before "A program which ...".

There are lots of places in this PR that don't use semantic line breaks. I don't think that should block the merge, but please bear it in mind for future edits, thanks!

Reply:
Looks like this has been addressed. Many thanks, @RobertLeahy!